hta_page.qmd

```diff
@@ -0,0 +1,22 @@
+---
+title: "HTA-R workstream"
+---
+
+**Lead**: Gregory Chen (MSD)
+
+**Objective**: 
+This workstream in SWE WG unites voluntary statistical software engineers that are experienced in HTA (Health Technology Assessment) or highly-interested in this area. 
+The team aims to develop open-source R tools of good quality in the right formats (R pkg, apps, user guides) that are useful to both manufacturers and payers, to support crucial analytic topics in HTA dossier submission across various countries, particularly the topics with unmet needs in R implementation and/or related to [EUnetHTA](https://www.eunethta.eu/).
```

Comment:
I would suggest not abbreviating "packages" with "pkg" here.

Reply:
adopted